### PR TITLE
FastAPI: Second steps. Trying to understand how this framework works.

### PR DESCRIPTION
**Pull request by goldenuni**

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1 @@
+WEATHER_API_KEY=YOUR_WEATHER_API_KEY
```
Comment (ArsenPidhoretskyi) [inline .env.sample]: Add empty line

File: city_app/crud.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+from sqlalchemy import select, insert, delete
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from . import models, schemas
+
+
+async def get_all_cities(db: AsyncSession) -> list[models.City]:
+    query = select(models.City)
+    city_list = await db.execute(query)
+    return [city[0] for city in city_list.fetchall()]
```
Comment (ArsenPidhoretskyi) [inline city_app/crud.py]: Use .scalars()

File: city_app/crud.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+async def get_city_by_id(
+        db: AsyncSession,
+        city_id: int
+) -> models.City | None:
+    query = select(models.City, models.City.id).where(
+        models.City.id == city_id
+    )
+    result = await db.execute(query)
+    city = result.scalars().first()
+    return city if city else None
```
Comment (ArsenPidhoretskyi) [inline city_app/crud.py]:
```suggestion
    return city

```
If statement is redundant

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Review by YukihiroSM — APPROVED** [review]
Looks nice!

**Pull request closed without merging**